config/locales/alchemy.de.yml

```diff
@@ -385,6 +385,7 @@ de:
     file: "Datei"
     file_rename_error: "Datei konnte nicht umbenannt werden."
     files: "Dateien"
+    from: 'von'
```

Comment:
Should these eventually be namespaced to something like `time_picker.from` or something similar? Again, this is not something I'd block upon.